lib/stat_tracker.rb

```diff
@@ -344,25 +358,13 @@ def most_tackles(season_id) #issue # 21 PASS
       end
     end
 
-    team_id_and_tackles_hash = {}
-    all_team_ids = []
-
-    games_in_season_hash[season_id.to_i][1]["team_id_and_tackles"].each do |pair|
-      all_team_ids << pair[0]
-    end
-
-    unique_team_ids = all_team_ids.uniq
-
-    unique_team_ids.each do |teamid|
-      team_id_and_tackles_hash[teamid] = 0
-    end
```

Comment:
Nice great to see that refactor we did was able to condense this many lines. It's looking great and you explained each part really well!